### PR TITLE
feat: add package 'jalv-select'

### PR DESCRIPTION
**Pull request by SpotlightKid**

### CODE REVIEW

File: nvchecker/archlinux-proaudio.toml [hunk 1]
```diff
@@ -11,6 +11,12 @@ include_regex = '^r(\d+)_(\d+)_(\d+)$'
 from_pattern = '^r(\d+)_(\d+)_(\d+)$'
 to_pattern = '\1.\2.\3'
 
+[jalv-select]
```
Comment (cbix) [inline nvchecker/archlinux-proaudio.toml:14]: This should go before jamulus, by lexical sorting ;)

File: nvchecker/old_ver.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "jamulus": "3.8.2",
+  "jalv-select": "1.3",
```
Comment (cbix) [inline nvchecker/old_ver.json:3]: Same here

File: packages/jalv-select/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+url='https://github.com/brummer10/jalv_select'
+license=('custom:Public Domain')
+depends=(gtkmm jalv)
+makedepends=(lilv serd sord sratom)
+source=("$pkgname-$pkgver.tar.gz::https://github.com/brummer10/${pkgname/-/_}/archive/v$pkgver.tar.gz")
+sha256sums=('13227087f107e35ecf51897ed63050c5ba6df1b442ffb2f7e876c46fb0fe11d7')
+
+prepare() {
+  cd ${pkgname/-/_}-${pkgver}
+  # Don't compress man pages, that's makepkg's job
```
Comment (cbix) [inline packages/jalv-select/PKGBUILD:18]: I see other packages also do this and thought this was fine. Is it considered bad practice in a Makefile?

File: packages/jalv-select/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+pkgname=jalv-select
+pkgver=1.3
+pkgrel=3
+pkgdesc='A small GUI to select installed LV2 plugins and run them with jalv.'
```
Comment (cbix) [inline packages/jalv-select/PKGBUILD:7]:
```suggestion
pkgdesc='A small GUI to select installed LV2 plugins and run them with jalv'
```

**Review by cbix — CHANGES_REQUESTED** [review]
All good, just some nits :)

File: packages/jalv-select/PKGBUILD (hunk 3 above)
Comment (SpotlightKid) [reply packages/jalv-select/PKGBUILD:18]: It led to non-reproducible build.

File: nvchecker/archlinux-proaudio.toml (hunk 1 above)
Comment (SpotlightKid) [reply nvchecker/archlinux-proaudio.toml]: Fixed with 21bd0a6.

File: nvchecker/old_ver.json (hunk 2 above)
Comment (SpotlightKid) [reply nvchecker/old_ver.json:2]: Fixed with 21bd0a6.

File: packages/jalv-select/PKGBUILD (hunk 4 above)
Comment (SpotlightKid) [reply packages/jalv-select/PKGBUILD]: Fixed with 21bd0a6.

File: packages/jalv-select/PKGBUILD (hunk 3 above)
Comment (SpotlightKid) [reply packages/jalv-select/PKGBUILD:18]: See here: https://ci.cbix.de/osam-cologne/archlinux-proaudio/329/1/9

**Review by cbix — APPROVED** [review]

**Pull request merged**